.travis.yml

```diff
@@ -17,7 +17,9 @@ jobs:
         - mypy --ignore-missing-imports . || true  # https://github.com/python/mypy/issues/7907
         - pytest --doctest-modules --ignore=project_euler/ --durations=10 --cov-report=term-missing:skip-covered --cov=. .
     - name: Project Euler
-      before_script: pip install pytest-cov
+      before_script:
+        - pip install pytest-cov pytest-subtests
+        - pytest --tb=no --no-summary --capture=no project_euler/validate_solutions.py  # fail fast on wrong solution
```

Comment:
Please put `|| true` at the end of this command and remove the draft status so that we can land this PR.  It will be good to expose this test process to more contributors.  We can enhance the capabilities in future pull requests.